modules/apt.sh

```diff
@@ -61,26 +62,33 @@ _apt_write_list_file() {
     local repo_url="$2"
 
     cat >"$repo_file" <<EOF
-deb https://${repo_url}/ubuntu ${SERIES} main
-# deb-src https://${repo_url}/ubuntu ${SERIES} main
+deb ${repo_url}/ubuntu ${SERIES} main
+# deb-src ${repo_url}/ubuntu ${SERIES} main
 EOF
 }
 
 _apt_add_auth() {
-    local repo_host="$1"
+    local repo_url="$1"
     local credentials="$2"
 
     local login="${credentials%:*}"
     local password="${credentials#*:}"
     [ -f "$APT_AUTH_FILE" ] || touch "$APT_AUTH_FILE"
     chmod 600 "$APT_AUTH_FILE"
-    echo "machine ${repo_host}/ubuntu/ login ${login} password ${password}" \
+    echo "machine ${repo_url/*:\/\//}/ubuntu/ login ${login} password ${password}" \
```

Comment:
Suggestion: use another local variable here, like in the awk code further down, called `repo_host_path`, or `repo_without_scheme` :), to make it more obvious what we are adding to the auth file. The sequence of `/` and `\` is never immediately obvious about what it does, and easy to typo.

Reply:
done